### DISCUSSION TITLE
Test this app over live store as custom app

### DISCUSSION THREAD

**Discussion by AminRafaey** [kinngh/shopify-node-mongodb-next-app#44] [Q&A]
Hi @kinngh Thank you for building such an informative repo and always thanks for the time you take to respond here. The repo works pretty well for the dev store. But what about the case if we want to test the app over the actual store as a custom app? Can we do this by staying in an existing repo? If yes the you can please give an overview how to achieve this?

**Comment by kinngh** [comment]
I've had a question from you before on this. What I recommend you to do is read the docs instead of asking around what to do. 

https://shopify.dev
